react.md

```diff
@@ -421,17 +421,289 @@ const SignupSchema = yup.object({
 - TODO: Docs/standards
 
 ## Styling
+We recognize that a one-size-fits-all approach to CSS isn't really possible and that overcomplicating styling for smaller apps can lead to unnecessary complexity. Knowing that, we make use of two approaches for styling: vanilla CSS, and CSS Modules.
+
+### Vanilla CSS
+CRA compiles `.css` files into a single style bundle using Webpack, and automatically runs your code through Autoprefixer. Please refer to our general CSS style guide for information on structuring and naming your selectors.
+
+If you find yourself writing a lot of CSS or getting heavily into complex BEM selectors, you might want to move along to CSS Modules.
+
 ### CSS Modules
-- TODO: Docs/standards
+While a traditional CSS approach can work well with simple React apps, we prefer making use of CSS Modules for styling more complex applications. CSS Modules support is built into CRA, all you have to do is use the `.module.css` extension instead of `.css`, and co-locate it with your React component. This tells Webpack a couple things:
 
-### Simple
-- TODO: What is a simple CSS use case? ie. I just need to style some stuff
-- TODO: postcss-icss-values for basic variable support without Sass, but use css vars if IE support isn't necessary
+1. Affix some additional text to your selectors post-compilation to make sure they're unique across your entire project. i.e. `.container` in `styles.modules.css` is transformed to something like `.styles__container_v23fn2`.
```

Comment:
thanks for including the example of before and after - `container` vs .`styles.....`
that helps _a lot_